x/mint/README.md

```diff
@@ -47,6 +46,56 @@ oneYear := time.Hour * 24 * 365
 | 19   | 1.50              |
 | 20   | 1.50              |
 
+## State
+
+See [./types/minter.go](./types/minter.go) for the `Minter` struct which contains all of the state of this module.
+
+## State Transitions
+
+The `Minter` struct is updated every block via `BeginBlocker`.
+
+## Begin Block
+
+See `BeginBlocker` in [./abci.go](./abci.go).
+
+## Events
+
+An event is emitted every time a block provision is minted. See `mintBlockProvision` in [./abci.go](./abci.go).
+
+## Client
+
+### CLI
+
+```shell
+$ celestia-appd query mint annual-provisions
+80235005639941.760000000000000000
+```
+
+```shell
+$ celestia-appd query mint genesis-time
+2023-05-09 00:56:15.59304 +0000 UTC
+
+```
+
+```shell
+$ celestia-appd query mint inflation
+0.080000000000000000
+```
+
+## Params
+
+All params have been removed from this module.
+
+## Future Improvements
+
+1. <https://github.com/celestiaorg/celestia-app/issues/1755>
+1. <https://github.com/celestiaorg/celestia-app/issues/1757>
+1. <https://github.com/celestiaorg/celestia-app/issues/1758>
```

Comment:
Not sure if this section is necessary. These are all minor improvements and it just increases the amount of overhead as we need to remember to remove them when we complete the issue